.buildkite/scripts/e2e/bootstrap.py

```diff
@@ -0,0 +1,150 @@
+        release_info = response.json()
+
+        download_urls = [asset["browser_download_url"] for asset in release_info["assets"]]
+        download_url = [url for url in download_urls if self.distro in url][0]
+        if download_url.strip() == '':
+            raise Exception(f"Could not resolve elastic-package distro download URL, release info: {release_info}.")
+
+        file_name = "downloaded_elastic_package_" + self.distro
+        # downloading with `urllib3` gives a different size file which causes a corrupted file issue
+        Util.run_or_raise_error(["curl", "-o", file_name, "--retry", "5", "--retry-delay", "5", "-fSL", download_url],
+                                "Failed to download elastic-package")
+        print("elastic-package is successfully downloaded.")
+
+        # Extract the downloaded tar.gz file
+        Util.run_or_raise_error(["tar", "zxf", file_name],
+                                f"Error occurred while unzipping {file_name}")
+
+    def __make_elastic_package_global(self) -> None:
+        Util.run_or_raise_error(["sudo", "mv", "elastic-package", "/usr/local/bin"],
+                                "Could not make `elastic-package` global")
+
+    def __clone_integrations_repo(self) -> None:
+        Util.run_or_raise_error(["retry", "-t", "5", "--", "git", "clone", "--single-branch",
+                                 "https://github.com/elastic/integrations.git"],
+                                "Error occurred while cloning an integrations repo. Check logs for details.")
+
+    def __get_profile_path(self) -> str:
+        return os.path.join(Util.get_home_path(), ".elastic-package/profiles/e2e")
+
+    def __create_config_file(self, sample_config_file: str, config_file: str) -> None:
+        with open(sample_config_file, "r") as infile, open(config_file, "w") as outfile:
+            for line in infile:
+                if "stack.logstash_enabled: true" in line:
+                    # Logstash is disabled by default, remove the comment
+                    line = line.lstrip('#').lstrip()
+                outfile.write(line)
+
+    def __setup_elastic_package_profile(self) -> None:
+        Util.run_or_raise_error(["elastic-package", "profiles", "create", "e2e"],
+                                "Error occurred while creating a profile. Check logs for details.")
+
+        print("`elastic-package` e2e profile created.")
+
+        # elastic-package creates a profile under home directory
+        config_example_file = os.path.join(self.__get_profile_path(), "config.yml.example")
+        config_file = os.path.join(self.__get_profile_path(), "config.yml")
+        self.__create_config_file(config_example_file, config_file)
+        Util.run_or_raise_error(["elastic-package", "profiles", "use", "e2e"],
+                                "Error occurred while creating a profile. Check logs for details.")
+
+    def __install_plugin(self) -> None:
+        with open("VERSION", "r") as f:
```

Comment:
Do we create this file somewhere?

```
    with open("VERSION", "r") as f:
         ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'VERSION'
```

Reply:
Hm., it is under the repo: https://github.com/elastic/logstash-filter-elastic_integration/blob/main/VERSION
Probably you are running the script inside the `.buildkite/scripts/e2e` (hidden part of the guide 🙈, will think re-organize/revise the guide).

Reply:
I have updated the `.buildkite/scripts/e2e/readme.md`.